Source/Engine/Engine.cs

```diff
@@ -1766,6 +1766,44 @@ private void DrawSpawns(Graphics g)
                         g.DrawLine(pPetChain, headX - offX, headY - offY, tailX, tailY);
                         DrawArrowHead(g, pPetChain, headX - offX, headY - offY, rads, 10f, 5f);
                     }
+                    if (m_drawPetLines && spawn.TargetIndex > 0 && m_game.Spawns.ContainsIndex(spawn.TargetIndex))
+                    {
+						if (spawn != m_game.Player)
```

Comment:
Please change the tabs in here to spaces (you should set up your editor to only use 4 spaces instead of tabs).